web/tests/functional/cppcheck/test_proj/divide_zero.cpp

```diff
@@ -0,0 +1,19 @@
+// -----------------------------------------------------------------------------
+//                     The CodeChecker Infrastructure
+//   This file is distributed under the University of Illinois Open Source
+//   License. See LICENSE.TXT for details.
+// -----------------------------------------------------------------------------
+
+// core.DivideZero (C, C++, ObjC)
+// Check for division by zero.
+#include "lib.h"
+
+void test1(int z) {
+  if (z == 0)
+    int x = 1 / z; // warn
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either the condition 'z==0' is redundant or there is division by zero at line 13.](https://app.codacy.com/app/CodeChecker/codechecker/pullRequest?prid=4623161)